src/reference/modules/usage/pages/variables.adoc

```diff
@@ -633,7 +631,7 @@ The resulting properties will be:
 
 Agents can define variables:
 
-* For the Linux agent, https://docs.cfengine.com/docs/3.15/reference-special-variables-sys.html[CFEngine system variable] are available.
+* For the classic (Unix) agent, https://docs.cfengine.com/docs/3.18/reference-special-variables-sys.html[CFEngine system variable] are available.
```

Comment:
We renamed "classic"  to "linux" in 7.2 https://rudderio.slack.com/archives/G97HW5U20/p1656330279100489?thread_ts=1656315436.355709&cid=G97HW5U20